host/cli/cli-add-command.go

```diff
@@ -0,0 +1,89 @@
+package cli
+
+import (
+	"crypto/sha256"
+	"encoding/base64"
+	"encoding/pem"
+	"errors"
+	"fmt"
+
+	host "github.com/flynn/flynn/host/types"
+	"github.com/flynn/flynn/pkg/cluster"
+	"github.com/flynn/go-docopt"
+)
+
+func init() {
+	Register("cli-add-command", runCliAddCommand, `
+usage: flynn-host cli-add-command
+
+Get the 'flynn cluster add' command to manage this cluster.`)
+}
+
+func runCliAddCommand(args *docopt.Args, client *cluster.Client) error {
+	hosts, err := client.Hosts()
+	if err != nil {
+		return fmt.Errorf("could not list hosts: %v", err)
+	}
+	if len(hosts) == 0 {
+		return errors.New("no hosts found")
+	}
+	var jobs []host.ActiveJob
+	for _, h := range hosts {
+		hostJobs, err := h.ListJobs()
+		if err != nil {
+			return fmt.Errorf("could not get jobs for host %v: %v", h.ID(), err)
+		}
+		for _, job := range hostJobs {
+			jobs = append(jobs, job)
+		}
+	}
+	var (
+		domain           string
+		key              string
+		pin              string
+		dashboardAppName string
+		dashboardDomain  string
+		dashboardToken   string
+	)
+	for _, job := range jobs {
+		app := job.Job.Metadata["flynn-controller.app_name"]
+		typ := job.Job.Metadata["flynn-controller.type"]
+		if app == "controller" && typ == "web" {
+			domain = job.Job.Config.Env["DEFAULT_ROUTE_DOMAIN"]
```

Comment:
The name is unfortunate, a job becomes "active" once it is accepted by a host, and then we continue to name it "active" even if it has stopped, so there could be multiple "active" controller web jobs from different releases and with different environment variables (i.e. some stopped, some running, some starting etc.).

We should just pick the most recent one.